### ISSUE TITLE
Linux Support 

### ISSUE THREAD

**Issue by FoxbitDreamtail** [baggyg/BendyVR#22]
For users like me who care about our privacy, Linux is the only secure option and unfortunately your mod does not seem to work with it, I have tried the windows version in the windows version of bendy and the ink machine on Linux using proton and that simply does not work, I can confirm that my Linux Mint 21.1 setup works with beat saber which is a Windows only game of course I have to use a legacy unity version but that shouldn't be a problem here as it's already running a legacy version of unity, and runs in desktop mode with no issue, the only issue in which I'm coming across is that it never starts in VR, I am using the steam VR beta and it just does not work, a Linux native version as bendy and the ink machine is Linux native would be nice.

**Comment by baggyg** [comment]
To be honest I am not sure why it wouldn't work with proton, unless Harmony has an issue patching .NET functions on Linux or somehow OpenVR was different there. Either way that's not something that's necessarily under my control. 

Does any of the other game mods work for you, I.e. Two Forks for Firewatch or Outer Wilds? 

If I had a precise error to work with I could possibly try a solution. The biggest issue is that I don't have a Linux machine to develop this on.

**Comment by FoxbitDreamtail** [comment]
The specific issue which I am coming across is apparently steam tries to launch desktop game theater and apparently desktop game theater simply doesn't exist apparently, at least not in the full release of steamvr for Linux, I don't know how someone would work around something like this but I wonder if it would be possible to make the my natively work with Linux in a way in which it shows up kind of like how source mods work if that's even possible, then you would have been in the machine and batim VR that you can just launch separately from within Steam after installing instead of having to worry about modifying files and whatnot, sorry for putting additional work on you it's just for some reason the Linux version of steamvr just does not contain "Desktop Game Theatre"

**Comment by FoxbitDreamtail** [comment]
Does it need to be disabled?

**Comment by FoxbitDreamtail** [comment]
Also, believe it or not one of my first instincts was to disable it, and no dice.

**Comment by baggyg** [comment]
Yes it should be disabled (although my mod should override that). 

So what's the error now, considering you've told steam not to launch Theatre mode?

**Comment by FoxbitDreamtail** [comment]
Actually I was talking from memory just then, at least from what I saw last, the truth of the matter is that me and my brother Share a singular quest as a VR headset and he is currently using that quest to play PC VR chat, I use that quest to buy alvr to play beat saber normally, so if possible, can we chat on Discord while we wait for my brother to get through "LPD Training"?

**Comment by FoxbitDreamtail** [comment]
I'm more than happy to send you a Discord link so we can chat.

**Comment by baggyg** [comment]
Feel free to drop onto the Team Beef Discord, where all of our ports are discussed. 

https://discord.gg/fA6m8SMZPA

However personally I'm not too active there as we are always working on new ports. Deep into Quake 4 at the moment.

**Comment by FoxbitDreamtail** [comment]
I see, and thanks for the link, I'll come back to this issue once the quest 2 gets recharged and I can reproduce the error.

**Comment by Patola** [comment]
@FoxbitDreamtail 2 years later...

This mod is working so perfectly right now that even bug #44 is the same as on Windows. I have a youtube channel dedicated for VR gameplays and mods on Linux and I have just published a video with the complete instructions on how to make it work on Linux: https://www.youtube.com/watch?v=2-WxZXhp-hY

**Comment by baggyg** [comment]
I suspect they didn't update Linux version which is why it works there.

**Comment by Patola** [comment]
> I suspect they didn't update Linux version which is why it works there.

I'm not using the Linux version but the Windows version via Proton. I show that at the start of the video, when I show the compatibility tab and Proton 9.0-4 is selected.
